parasoft-findings-teamcity-agent/src/main/java/com/parasoft/findings/teamcity/agent/ParasoftFindingsBuildProcess.java

```diff
@@ -161,7 +163,7 @@ private void doWork() {
         String settingsPath = params.get(SETTINGS_LOCATION);
         Properties properties = _localSettingsHelper.loadLocalSettings(checkoutDir, settingsPath);
         if (!properties.isEmpty()) {
-            if (_localSettingsHelper.isDtpUrlValidForTeamCity(properties.getProperty("dtp.url"))) {
+            if (_localSettingsHelper.isDtpUrlValidForTeamCity(properties.getProperty(DTP_URL))) {
```

Comment:
Can you change `DTP_URL` to `LocalSettingsHelper.DTP_URL` and remove the import?